qemu/tests/blockdev_inc_backup_modify_backing_bitmaps.py

```diff
@@ -11,8 +11,9 @@ class BlkIncModifyBackingBitmaps(BlockDevSnapshotTest):
     def reopen_backing_image(self, node_name):
         opts = []
         fmt_node = self.main_vm.devices.get_by_qid(node_name)[0]
-        file = fmt_node.get_param("file")
-        item = {"driver": "qcow2", "node-name": node_name, "file": file,
+        file_node = fmt_node.get_param("file")
+        driver = fmt_node.get_param("driver")
+        item = {"driver": driver, "node-name": node_name, "file": file_node,
                 "read-only": False}
         qemu_binary = utils_misc.get_qemu_binary(self.params)
         qemu_version = utils_qemu.get_qemu_version(qemu_binary)[0]
```

Comment:
Since raw format don't support persistent bitmap, we can delete 'persistent' in the steps. 

 74     2) add a persistent bitmap
 75     3) create snapshot target node
 76     4) do snapshot to target node
 77     5) add a persistent bitmap to snapshot node
 78     6) reopen backing image by (x-)blockdev-reopen
 79     7) remove all persistent bitmaps